### DISCUSSION TITLE
关于对微软网络连接问题的讨论

### DISCUSSION THREAD

**Discussion by LokerL** [LokerL/tts-vue#83] [Ideas]
有用户私信我或提出issues：
> 你好，我特意注册一个号来给你说，有时候无法转换语音，这个不是你软件的问题，是国内有时候屏蔽了微软TTS的服务导致的，遇到这种时候，用全局nat模式的代理是可以使用的。

>感谢贡献这个TTS-VUE，非常好用。我自己的一点小发现，网络连接问题或许和微软的IP限制有关。网页端试用超过某个限制（或许是次数，或许是字符数）以后怎么刷新都是连不上的，换一个IP就能立刻连接上。

>我测试发现网络问题 应该跟微软的限制有关。websocket的访问类似于请求限制之类的。
>一般发生在多次不规则的请求中 比如 测试语音一次 并没有完整接受完数据时又点击请求类似于这样 操作后就会连不上
>重新换ip 重启路由器以后就解决了。

至于以上建议是否有帮助，可以自行辨别

**Comment by CholeDell** [comment]
魔法上网好像能解决限制😎

**Comment by CHEN6311** [comment]
作者免费的软件真的不要吗？参考一下他的代码呗？

**Reply by LokerL** [reply]
> 作者免费的软件真的不要吗？参考一下他的代码呗？

已更新
